catalog.html

```diff
@@ -230,7 +234,7 @@ <h2>Подпишитесь на рассылку</h2>
       </li>
     </ul>
     <a class="footer-contacts-address-phone" href="tel:+78128121212">+7 (812) 812-12-12</a>
-    <img class="footer-logo" src="https://htmlacademy.ru/" alt="htmlacademy.">
+    <img class="footer-logo" src="images/bottom/logo_htmlacademy .svg" alt="htmlacademy.">
```

Comment:
```suggestion
    <img class="footer-logo" src="images/bottom/logo-htmlacademy.svg" alt="htmlacademy.">
```

все картинки должны называться через дефис

Reply:
в индекса тоже поправь